docs/RESOURCES.md

```diff
@@ -10,6 +10,7 @@
 * [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
 * [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
 * [@dotnet](https://twitter.com/DotNet) is the official .NET Twitter account.
+* [Gitter](https://gitter.im/exercism/xcsharp) is the C# Gitter room.
```

Comment:
I referenced https://gitter.im/exercism/xcsharp in my change, I am assuming this is correct.     There isn't any traffic over there yet which is why I mention it (could just be no one knows its there yet).     If I got this wrong though, let me know.    I believe this is right, so will go ahead and submit PR